otter/grade/__init__.py

```diff
@@ -103,6 +106,9 @@ def main(
     if ext not in _ALLOWED_EXTENSIONS:
         raise ValueError(f"Invalid submission extension specified: {ext}")
 
+    if result_queue:
+        loggers.add_queue_handler(result_queue)
```

Comment:
Since the logging handler is created here, `result_queue` doesn't need to be passed anywhere in containers.py, right?